tests/Sniffs/Classes/RequireOneLinePropertyDocCommentSniffTest.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+class RequireOneLinePropertyDocCommentSniffTest extends \SlevomatCodingStandard\Sniffs\TestCase
+{
+
+	public function testNoErrors(): void
+	{
+		self::assertNoSniffErrorInFile(self::checkFile(__DIR__ . '/data/oneLinePropertyDocCommentNoErrors.php'));
+	}
+
+	public function testErrors(): void
+	{
+		$report = self::checkFile(__DIR__ . '/data/oneLinePropertyDocCommentErrors.php');
+
+		self::assertSame(9, $report->getErrorCount());
+
+		self::assertSniffError($report, 6, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 12, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 18, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 24, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 29, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 35, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 41, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 49, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+		self::assertSniffError($report, 53, RequireOneLinePropertyDocCommentSniff::CODE_MULTI_LINE_PROPERTY_COMMENT);
+
+		self::assertAllFixedInFile($report);
```

Comment:
@kukulich This does not trigger the `$fix` in the sniff itself (it's `false`), what did I miss? :/

Reply:
No idea, will debug after the requested changes.

Reply:
It's strange that the lines are covered in code coverage.